main.go

```diff
@@ -410,3 +414,17 @@ func mustRunHealthz() {
 		panic(err)
 	}
 }
+
+func ReadSubnetFromSubnetFile(path string, config *etcdv2.EtcdConfig) {
```

Comment:
If you're adding a new library for reading the file, why not use the same library for writing the file?